crates/serpen/tests/fixtures/comprehensive_ast_rewrite/main.py

```
@@ -0,0 +1,103 @@
+    db_result = db_process("database_data")
+    util_result = process("utility_data")  # Imported function
+    auth_result = auth_process("auth_data")
+
+    # Using imported classes with conflicts
+    util_logger = UtilLogger("util")
+    model_logger = Logger("model")  # Different Logger class
+
+    # Using imported User classes vs local User class
+    auth_user = User("auth_type")  # Local User class
+    model_user = UserModel("model_type")  # Imported User class
+    service_user = User("service_type", "password")  # Imported User class from auth
```

Pylint (3.3.7)

[error] Too many positional arguments for constructor call

(E1121)